### DISCUSSION TITLE
Buscando Equipo Hackaton

### DISCUSSION THREAD

**Discussion by AnaCatherineC** [Launch-X-Latam/MisionFrontEnd#557] [Team Posts]
Hola a Todos aun no tengo equipo para el Hackaton ,estoy buscando quien quiera unirse o me acepte en su equipo n.n ,Saludo a  todos !!

**Comment by miller-vrg** [comment]
Me faltan dos si quieres únete, dame tu correo

**Comment by rubenmantilladev** [comment]
Yo también quisiera unirme. Mi correo es ***@***.***

El 12 nov. 2022 18:24, Miler Vargas Mola ***@***.***> escribió:

Me faltan dos si quieres únete, dame tu correo

**Comment by miller-vrg** [comment]
Listo estamos competo entonces

**Comment by rubenmantilladev** [comment]
Nos comunicaremos por este medio o ya hay algo formado?

El 12 nov. 2022 19:10, Miler Vargas Mola ***@***.***> escribió:

Listo estamos competo entonces

**Comment by AnaCatherineC** [comment]
mi correo es lx14959@innovaccion.mx :D

**Comment by Max80318** [comment]
Me agregan ***@***.*** <http://innovaccion.mx/>*

**Comment by AbnerHub** [comment]
Hola me podrían agregar mi correo es lx02522@innovaccion.mx

**Comment by innovategm** [comment]
aun les falta integrante? 
lx05800@innovaccion.mx
